src/main/java/ai/elimu/web/content/word/WordCreateController.java

```diff
@@ -74,13 +74,13 @@ public String handleRequest(Model model, @RequestParam(required = false) String
         if (StringUtils.isNotBlank(autoFillText)) {
             word.setText(autoFillText);
             
-            autoSelectLetterSoundCorrespondences(word);
+            autoSelectLetterSounds(word);
             // TODO: display information message to the Contributor that the letter-sound correspondences were auto-selected, and that they should be verified
         }
         
         model.addAttribute("word", word);
         model.addAttribute("timeStart", System.currentTimeMillis());
-        model.addAttribute("letterSoundCorrespondences", letterSoundDao.readAllOrderedByUsage()); // TODO: sort by letter(s) text
+        model.addAttribute("letterSounds", letterSoundDao.readAllOrderedByUsage()); // TODO: sort by letter(s) text
```

Comment:
The creation of a new Word at http://localhost:8080/webapp/content/word/create stopped working, since the letter-sound drop-down is empty. Looks like you need to update the frontend file for this controller as well.

<kbd>![](https://github.com/user-attachments/assets/55f51c3f-f78c-40af-a340-b2482d49941b)</kbd>